vumi_http_proxy/test/test_clickme.py

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+from click.testing import CliRunner
+from vumi_http_proxy import clickme
+from twisted.trial import unittest
+
+# Testing segment
+
+
+class TestClickMe(unittest.TestCase):
+    def test_click(self):
+        runner = CliRunner()
+        clickme.test = True
+        result = runner.invoke(clickme.cli)
+        assert 'Starting connection to 0.0.0.0:8080' in result.output
```

Comment:
We should use `self.assertEqual` here and maybe assert that this is the only line of output. We should also assert that the `exit_code` is 0.